airflow/providers/databricks/hooks/databricks.py

```diff
@@ -407,6 +407,14 @@ def get_run_state(self, run_id: str) -> RunState:
         """
         Retrieves run state of the run.
 
+        Please note that any Airflow tasks that call the ``get_run_state`` method will result in
+        failure unless you have enabled xcom pickling.  This can be done using the following
+        environment variable: ``AIRLFOW_CORE_ENABLE_XCOM_PICKLING=TRUE``
```

Comment:
Is this the correct variable? I think you need double underscores?

Reply:
you're right. FIxed